Apps/W1/ShopifyConnector/app/Transactions/Tables/ShpfyPaymentMethodMapping.Table.al

```diff
@@ -0,0 +1,51 @@
+/// <summary>
+/// Table Shpfy Payment Method Mapping (ID 30134).
+/// </summary>
+table 30134 "Shpfy Payment Method Mapping"
+{
+    Access = Internal;
+    Caption = 'Shopify Payment Method';
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; "Shop Code"; Code[20])
+        {
+            Caption = 'Shop Code';
+            DataClassification = SystemMetadata;
+            TableRelation = "Shpfy Shop";
+        }
+        field(2; Gateway; Text[30])
+        {
+            Caption = 'Gateway';
+            DataClassification = CustomerContent;
+            TableRelation = "Shpfy Transaction Gateway";
+        }
+        field(3; "Credit Card Company"; Text[30])
+        {
+            Caption = 'Credit Card Company';
+            DataClassification = CustomerContent;
+            TableRelation = "Shpfy Credit Card Company";
+        }
+        field(4; "Payment Method"; Code[10])
```

Comment:
please rename field to: "Payment Method Code"

Reply:
Updated with commit "Object Renaming"